README.md

```diff
@@ -28,9 +60,55 @@ To enable extra development features please run `docker-compose --profile dev up
+You can access `phpmyadmin` by visiting `http://localhost:8080`.
+
 
 ### Rebuilding the database
 
 To rebuild the database from scratch and apply again all the updates run  `docker-compose up --renew-anon-volumes sql`. Note: this will WIPE any custom handmade changes, including accounts.
+
+## Client Setup
+
+You will need to download or create file `reamlist.wtf` and place it where `WoW.exe` is located, this is what's used to set the server IP to be able to connected to the realm(s).
+An example how it should look: `SET realmlist "172.25.176.1"`
+
+After that you will need to create file `wow.ses` and the first ane second line of the file will contain the `username` then `password` for your login, depening on your server settings, this information can be or not automatically created. 
+An example how it should look: 
+`user`
+`user`
+
+To be able to launch client (that hasn't not been modified or altered) you will need need to start `WoWClient.exe` with `-uptodate` parametre and **highly recommended** to use `-windowed"`, the easiest way it's to do this via a windows **bat**ch file, an example of how it should look:
+`start WoWClient.exe -uptodate -windowed`.
+
+Optionally you may want to create the cache folder by adding `Rmdir /S "WDB"` before `start` command.
+
+In-game you may or need to select te `Change Realm` button to be able to login into your realm.
+
+When you login for the first time is recommended that you run the command `pwdchange` it will tell how it's supposed to be executed after, if you run more than 1 realm, keep in mind that the `user` and `password` are per realm, there's no "shared auth server" currently.
+
+
+## Commun Issues
+
+Can't start up the docker container or database service, example of text (this case docker):
+`Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:3306 -> 0.0.0.0:0: listen tcp 0.0.0.0:3306: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.`
+Make sure the `port` used for `MariaDB` is not being used by another `MariaDB`, `MySQL` or similiar services.
+
+I get `Invalid realm list` assumed you've set the correct information in `config.yml` the server has probably not fully started yet.
+You will see something similiar in your `world` terminal: `2025-08-01 01:11:25 [INFO] [01/08/2025 01:11:25] Alpha Core is now running.` this means is ready to be logged into.
+
+> [!IMPORTANT]  
+> Keep in mind the client being old and being an experimental build, you will no find much stability or perfomance, this has nothing to do with the core, it's a client exclusive issue.
+
+## Disclaimer
+
+The `Alpha Project` does not distribute a client. You will need to find your own clean `0.5.3` client on the internet.
+
+The `Alpha Project` Team and Owners DO NOT in any case sponsor nor support illegal public servers. If you use these projects to run an illegal public server and not for testing and learning it is your personal choice.
```

Comment:
Running a private server is not illegal in most countries, so I prefer to use the `rogue` or `unofficial` adjectives or similar instead of `illegal`.